pom.xml

```diff
@@ -13,7 +13,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.curityVersion>6.6.0</project.curityVersion>
+        <project.curityVersion>7.0.0</project.curityVersion>
         <project.slf4jVersion>1.7.32</project.slf4jVersion>
```

Comment:
The provided version currently is 1.7.36

Reply:
Updated